src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -176,12 +177,6 @@ where
 
     let reclock_follower = ReclockFollower::new(config.as_of.clone());
 
-    let (resume_tx, resume_rx) = config.metrics.get_instrumented_source_channel(
```

Comment:
are there any metrics we can include to track progress of the new operator?

Reply:
Added metrics for the binding buffers in the operator